src/components/Survey/SurveysHistory/SurveysHistoryView.tsx

```diff
@@ -44,7 +44,7 @@ const SurveyHistoryElement: React.FC<SurveyHistoryElementProps> = ({
   );
 
   const sortedSurveys = surveys.sort(
-    (a, b) => dayjs(b.updatedAt).unix() - dayjs(a.updatedAt).unix()
+    (a, b) => getUnixTime(b.updatedAt) - getUnixTime(a.updatedAt)
```

Comment:
Wouldn't this work without converting to unix time?
![obraz](https://user-images.githubusercontent.com/12448522/112280340-34d75b80-8c85-11eb-922a-166b2e71328f.png)

Reply:
@v0idzz Yes, it would work. I replaced this with `date-fns` function, as I only focused on formatting using native Date functions. However, since here is simplifies the code even more, let's replace this as well.